website/docs/integrations/hive/configuration/hive_conf.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 2
+title: Configuration parameters
+---
+:::info
+This list doesn't include transport configuration parameters, see [Transport](transport.md)
+:::
+## Configuration
+
+The following parameters can be specified:
```

Comment:
nit: maybe it should be explicitly mentioned that all other OpenLineage client configuration options are supported here as well, with `hive.openlineage.` prefix instead of `openlineage.`

Reply:
done